docs/csi_ug_config_enable_snapshot.md

```diff
@@ -0,0 +1,39 @@
+# Creating a VolumeSnapshotClass
```

Comment:
This MD file refers to creating a volume snapshot class, but the file name refers to enabling the snapshot.
Since other `csi_ug_config_create...` files have the same title in their name, I suggest renaming this file to:
`csi_ug_config_create_volumesnapshotclass.md`

Reply:
updated to csi_ug_config_create_vol_snapshotclass

Reply:
I suggest using the whole word instead of abbreviations:
`csi_ug_config_create_volume_snapshotclass`

Reply:
updated to csi_ug_config_create_vol_snapshotclass.md

Reply:
Why do you prefer to keep the abbreviation?

Reply:
nvm, answered in a different conversation